### PR TITLE
Port to py3, breaking changes

### PR DESCRIPTION
**Pull request by aanil**
Will be installed on the py3 env on irma.

### CODE REVIEW

File: ngi_reports/ngi_reports.py [hunk 1]
```diff
@@ -60,11 +60,11 @@ def make_reports (report_type, working_dir=os.getcwd(), config_file=None, **kwar
     # Get parsed markdown and print to file(s)
     LOG.debug('Converting markdown to HTML...')
     output_mds = report.parse_template(template)
-    for output_bn, output_md in output_mds.iteritems():
+    for output_bn, output_md in list(output_mds.items()):
         try:
             # Ignore special character in the sring, panda might fail because of that
-            output_md = output_md.encode('ascii', 'ignore')
-            with open('{}.md'.format(output_bn), 'w') as fh:
+            #output_md = output_md.encode('ascii', 'ignore')
```
Comment (ssjunnebo) [inline ngi_reports/ngi_reports.py:66]: Just to double check, do you want to keep this commented out?

File: ngi_reports/reports/__init__.py [hunk 2]
```diff
@@ -106,16 +106,16 @@ def _getfcid(str):
...
-        Flattens a dict by concatinating the keys
-        seperated by the specified separator (sep) until it finds a value.
+        Flattens a dict by concatenating the keys
+        separated by the specified separator (sep) until it finds a value.
         Optionally it adds parent_key as a prefix.
         E.g. {my : {nested: {key: value}}} -> "my.nested.key: value"
         """
         items = []
-        for k, v in d.iteritems():
+        for k, v in list(d.items()):
```
Comment (ssjunnebo) [inline ngi_reports/reports/__init__.py:115]: Strictly speaking, you don't need list() when you're just iterating over things, but it's fine to keep it if you want 🙂

**Review by ssjunnebo — APPROVED** [review]
Looks good! Great work! 😄

File: ngi_reports/ngi_reports.py (hunk 1 above)
Comment (aanil) [reply ngi_reports/ngi_reports.py]: Done!

File: ngi_reports/reports/__init__.py (hunk 2 above)
Comment (aanil) [reply ngi_reports/reports/__init__.py]: Done!

**Review by ssjunnebo — APPROVED** [review]

**Pull request merged**